docs/source/tutorial/run_cluster.md

```diff
@@ -0,0 +1,35 @@
+# Running DeepReg on the Cluster
+This tutorial gives an example of how to run DeepReg remotely (e.g. on a cluster).  Our example is specific to the UCL cluster, which has operating system CentOS 7 (similar to Ubuntu), with job scheduler Sun Grid Engine (SGE).  More information on the specific configuration at UCL is available [here](https://hpc.cs.ucl.ac.uk/job-submission/).
+
+# Installing the Environment
+Install the environment in the cluster, as described [here](../../../README.md).  In the case you do not have root access (as the case of the UCL cluster), you might need to use pip -u option to install the requirements.txt file.
+
+# Example Script
+Below is the submission script for running quick start example [here](../getting_started/quick_start.md).  Change `<DeepReg_dir>` in the script to the remote DeepReg repo location and save the below code in a `<your_name>.qsub`.  Submit the job with `qsub <your_name>.qsub` and check the status of the job with `qstat`, the saved stdout and stderr is in `<DeepReg_dir>/logs/`.
+
+```
```

Comment:
add a bash after "```" might provide better coloring of the markdown.

Reply:
Apologies, I don't understand your comment

Reply:
oh sorry,

when you push code, you did like
```
docker run --name <container_name> --privileged=true -ti deepreg bash
```

but if you add the language of code after the first "```", you can have better coloring.

```bash
docker run --name <container_name> --privileged=true -ti deepreg bash
```

<img width="612" alt="image" src="https://user-images.githubusercontent.com/4066929/95673689-e33d8780-0ba2-11eb-895f-2f44cc249e6f.png">

Reply:
I didn't use docker... I just pushed the branch from my local machine to github.

Reply:
![image](https://user-images.githubusercontent.com/4066929/95685302-dba3d000-0bee-11eb-93ec-b97c3614c839.png)

I mean we need to add this `bash` after the three quote signs

Reply:
thanks for clarifying

Reply:
The script is SGE though
#$ is a comment in bash, but here it is an environment variable
the bash script starts after